soccer/src/soccer/planning/planner/settle_planner.cpp

```diff
@@ -536,4 +536,18 @@ void SettlePlanner::reset() {
     previous_ = Trajectory{};
 }
 
+bool SettlePlanner::is_done() const {
+    // FSM: Intercept -> Dampen
+    // (see process_state_transition())
+    if (current_state_ != SettlePlannerStates::Dampen) {
+        return false;
+    }
+
+    // Dampen is done when the ball is slow-ish, let collect handle actually
+    // taking possession
+    // TODO(Kevin): consider making settle + collect one planner
+    double SETTLE_BALL_SPEED_THRESHOLD = 0.75;
```

Comment:
hmm. I can fix all the style issues or let the warnings persist. what do you think? 

the reason I didn't make any of them into ROS params already is because I don't love our existing custom param server and wanted to experiment with using default ROS params first.

Reply:
referenced in clickup card https://app.clickup.com/t/31g2yfy